packages/venia-concept/.graphqlconfig

```diff
@@ -0,0 +1,12 @@
+{
+    "projects": {
+        "venia": {
+            "schemaPath": "lastCachedGraphQLSchema.json",
+            "extensions": {
+                "endpoints": {
+                    "default": "${env:MAGENTO_BACKEND_URL}graphql"
+                }
```

Comment:
```suggestion
                },
                "validate-queries": {
                    "filesGlob": "src/**/*.{js,graphql,gql}",
                    "clients": ["apollo", "literal"]
                }
```

Adding this allows you to pull the config out of context instead of having a long command in package scripts.